files/common/var/lib/delphix-platform/ansible/10-delphix-platform/roles/delphix-platform/tasks/main.yml

```diff
@@ -73,6 +73,40 @@
     state: directory
     mode: 01777
 
+#
+# Restrict cron permissions. All jobs are owned by root so there's no
+# reason to allow others any level of access.
+#
+- file:
+    path: /etc/crontab
+    state: file
+    mode: 0600
+
+- file:
+    path: /etc/cron.d
+    state: directory
+    mode: 0700
+
+- file:
+    path: /etc/cron.hourly
+    state: directory
+    mode: 0700
+
+- file:
+    path: /etc/cron.daily
+    state: directory
+    mode: 0700
+
+- file:
+    path: /etc/cron.weekly
+    state: directory
+    mode: 0700
+
+- file:
+    path: /etc/cron.monthly
+    state: directory
+    mode: 0700
```

Comment:
I think you can combine all of these using the `{{ item }}` construct. Something like this:
```
- file:
    path: "{{ item }}"
    state: directory
    mode: 0700
  with_items:
    - /etc/cron.d
    - /etc/cron.hourly
    - /etc/cron.daily
    - /etc/cron.weekly
    - /etc/cron.monthly
```
There's an example like this where we delete multiple files at once for GCP at line 403-ish.

Reply:
Neat!